### ISSUE TITLE
Command should use a wrapped sendto, so we can log in one place

### ISSUE THREAD

**Issue by lidaobing** [iptux-src/iptux#239]
```
$ git grep sendto
src/iptux/Command.cpp:    sendto(sock, buf, size, 0, (struct sockaddr *)&addr, sizeof(addr));
src/iptux/Command.cpp:      sendto(sock, buf, size, 0, (struct sockaddr *)&addr, sizeof(addr));
src/iptux/Command.cpp:  sendto(sock, buf, size, 0, (struct sockaddr *)&addr, sizeof(addr));
src/iptux/Command.cpp:  sendto(sock, buf, size, 0, (struct sockaddr *)&addr, sizeof(addr));
src/iptux/Command.cpp:  sendto(sock, buf, size, 0, (struct sockaddr *)&addr, sizeof(addr));
src/iptux/Command.cpp:  sendto(sock, buf, size, 0, (struct sockaddr *)&addr, sizeof(addr));
src/iptux/Command.cpp:    sendto(sock, buf, size, 0, (struct sockaddr *)&addr, sizeof(addr));
src/iptux/Command.cpp:  sendto(sock, buf, size, 0, (struct sockaddr *)&addr, sizeof(addr));
src/iptux/Command.cpp:  sendto(sock, buf, size, 0, (struct sockaddr *)&addr, sizeof(addr));
src/iptux/Command.cpp:  sendto(sock, buf, size, 0, (struct sockaddr *)&addr, sizeof(addr));
src/iptux/Command.cpp:  sendto(sock, buf, size, 0, (struct sockaddr *)&addr, sizeof(addr));
src/iptux/Command.cpp:  sendto(sock, buf, size, 0, (struct sockaddr *)&addr, sizeof(addr));
src/iptux/Command.cpp:  sendto(sock, buf, size, 0, (struct sockaddr *)&addr, sizeof(addr));
src/iptux/Command.cpp:  sendto(sock, buf, size, 0, (struct sockaddr *)&addr, sizeof(addr));
```

**Comment by lidaobing** [comment]
ref #217

**Comment by lidaobing** [comment]
fixed in #240
